mlflow/store/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Constants shared by several backend store implementations.
+"""
+
+DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH = "./mlruns"
```

Comment:
Maybe a quick comment on what this is used for (the name is pretty good, but maybe we could contextualize 'default' a bit).